color_refinement.py

```diff
@@ -215,25 +219,73 @@ def get_number_automorphisms(g: Graph) -> int:
     return get_number_isomorphisms(g, g.deepcopy(), True)
 
 
+def store_isomorphism(i: int, j: int, known_isomorphisms: Dict[int, Set[int]]):
```

Comment:
Wat deze methode doet: alle bekende isomorfismen van `i` en `j` worden opgeslagen voor al deze isomorfismen. Er wordt niet opgeslagen dat een graaf een isomorfisme is met zichzelf.

Reply:
Ik denk dat ik niet snap wat je hier mee bedoelt...